src/panels/lovelace/editor/config-elements/hui-sensor-card-editor.ts

```diff
@@ -143,6 +163,14 @@ export class HuiSensorCardEditor
         return this.hass!.localize(
           "ui.panel.lovelace.editor.card.sensor.graph_type"
         );
+      case "min":
+        return this.hass!.localize(
+          "ui.dialogs.helper_settings.input_number.min"
+        );
+      case "max":
+        return this.hass!.localize(
+          "ui.dialogs.helper_settings.input_number.max"
+        );
```

Comment:
Set a new translation key for this card and do not reuse one from the input number dialogs

Reply:
Done in [`1f22a7f`](https://github.com/home-assistant/frontend/pull/24358/commits/1f22a7f600866aae8857765db284f15caee26c7a).